Abdullah_AlQahtani/week_04/day_02/MTA_2_Ruby/MTA.rb

```diff
@@ -0,0 +1,45 @@
+$lineN = ["Times Square", "34th", "28th", "23rd", "Union Square", "8th"]
+$lineL = ["8th", "6th", "Union Square", "3rd", "1st"]
+$line6 = ["Grand Central", "33rd", "28th", "23rd", "Union Square", "Astor Place"]
+
+def getLine(line)
+  array = []
+  case line
+  when "N" then array = $lineN
+  when "L" then array = $lineL
+  when "6" then array = $line6
+  end
+  array
+end
+
+def get_trip_string(line, startStop, endStop)
+  lineArray = getLine(line);
+  stopsString = ""
+
+  startIndex = lineArray.index(startStop);  # starting point index
+  endIndex = lineArray.index(endStop);  # ending point index
+
+  if startIndex < endIndex
+    for i in ((startIndex + 1)..endIndex)
+      stopsString += lineArray[i] + ", "
+    end
+  else
+    for i in (startIndex..(endIndex - 1))
+      stopsString += lineArray[i] + ", "
+    end
+  end
+  stopsString
```

Comment:
This is ok, though I think it's a good idea to to try out different loops, `for` tends to be a bit messier in Ruby.